flask_fixtures/__init__.py

```diff
@@ -137,12 +143,47 @@ def load_fixtures(db, fixtures):
                 obj = model(**fields)
                 db.session.add(obj)
             db.session.commit()
+            # store the table associated with this model for future cleanup
+            _fixture_tables.append(model.__table__)
         elif 'table' in fixture:
             table = Table(fixture['table'], metadata)
             conn.execute(table.insert(), fixture['records'])
+            _fixture_tables.append(table)
         else:
             raise ValueError("Fixture missing a 'model' or 'table' field: {0}".format(json.dumps(fixture)))
 
+def delete_fixtures(db):
+    """Deletes the loaded fixtures from database
+    """
+    truncate_db(db, _fixture_tables)
+    del _fixture_tables[:]
+
+def truncate_db(db, fixture_tables=[]):
+    # we can have multiple binds, get the tables and engines with each table
```

Comment:
doesn't `_fixture_tables` have to be cleared in this case as well?
Do not use mutable default arg, it is also module scoped (pretty much global)

Reply:
if `truncate_db` is never called without fixture_tables, remove the default arg.

Reply:
`truncate_db` is called without `fixture_tables` when `truncate_db` set to `True` in test class

Reply:
If you need the default arg use a tuple (immutable) and not a list (mutable)
https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments